include/aspect/postprocess/geoid.h

```diff
@@ -118,9 +123,24 @@ namespace aspect
         double density_below;
 
         /**
-         * A parameter to control whether to include the dynamic topography contribution on geoid
+         * A parameter to control whether to include the surface topography contribution on geoid
+         */
+        bool include_surface_topo_contribution;
+
+        /**
+         * A parameter to control whether to include the CMB topography contribution on geoid
+         */
+        bool include_CMB_topo_contribution;
+
+        /**
+         * A parameter to specify if the top boundary is an active free surface
+         */
+        bool use_real_surface_topography;
```

Comment:
I would suggest changing this variable name to 'use_free_surface_topography' and below 'use_free_CMB_topography'. "Real" sounds to me like it's Earth-like, which it doesn't have to be so maybe 'free' is slightly more descriptive.